zndraw/zndraw.py

```diff
@@ -118,19 +123,25 @@ def __delitem__(self, index):
 
     def insert(self, index, value):
         """Insert atoms before index"""
-        self.socket.emit("atoms:insert", {index: atoms_to_json(value)})
+        if isinstance(value, ase.Atoms):
+            value = Frame.from_atoms(value)
+        self.socket.emit("atoms:insert", {index: value.frame_to_json()})
 
-    def append(self, value: ase.Atoms) -> None:
+    def append(self, value) -> None:
```

Comment:
please add back all the type hints that have been removed.

Reply:
Would it be alright to use t.Union for this? As we can have either `Frame` or `ase.Atoms` as values.

Reply:
Yes